composer.json

```diff
@@ -7,8 +7,18 @@
         "php": ">=5.4.0",
         "react/promise": "~2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.5"
```

Comment:
Super curious why this would be omitted. It reduces friction for new contributors and is ignored when this repository is consumed as a dependency...